### PR TITLE
Feature/104

### PR DESCRIPTION
**Pull request by Haris188**

### CODE REVIEW

File: web-client/src/pages/api/vtmembers/mockMember.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import {MembersListItem} from '../../../shared/redux/volunnteer-toronto/members'
```
Comment (l2silver) [inline web-client/src/pages/api/vtmembers/mockMember.ts:1]: Nice job with the API mock!

File: web-client/src/pages/volunteer-toronto/members.tsx [hunk 2]
```diff
@@ -14,8 +14,10 @@ const VolunteerTorontoMembers: NextPage = ()=>{
                 <title>Voluteer Toronto Members</title>
             </Head>
             <Container>
-                <SearchBar />
-                <Btns />
+                <Box display="flex" justifyContent="space-between" alignItems="center">
```
Comment (l2silver) [inline web-client/src/pages/volunteer-toronto/members.tsx:17]: Good idea

File: web-client/src/shared/components/volunteer-toronto/members/Btns/index.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+
+import React from 'react'
+import { Button } from '@material-ui/core'
+
+const Btns: React.FunctionComponent = ()=>{
+    const createOrganization = ()=>{
+        // implement
```
Comment (l2silver) [inline web-client/src/shared/components/volunteer-toronto/members/Btns/index.tsx:7]: Nice

File: web-client/src/shared/components/volunteer-toronto/members/MemberTable/ButtonGroup.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    return (
+        <div>
+            <Button onClick={sendResetPassRequest}>
+                Reset Password
+            </Button>
+            <Button color="secondary" onClick={sendDeleteUserRequest}>
+                Delete
+            </Button>
+            <Button color="primary" onClick={navigateToEmailCompose}>
+                <Link href="mailto:example@email.com">Email</Link>
```
Comment (l2silver) [inline web-client/src/shared/components/volunteer-toronto/members/MemberTable/ButtonGroup.tsx:35]: Looking good

File: web-client/src/shared/components/volunteer-toronto/members/MemberTable/index.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
+
```
Comment (l2silver) [inline web-client/src/shared/components/volunteer-toronto/members/MemberTable/index.tsx:1]: Very clean

File: web-client/src/shared/redux/volunnteer-toronto/members/index.ts [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+export const filterMembers = (filterString: string):MemberThunk =>{
+    return (dispatch, getState)=>{
+        const state = getState().volunteerTorontoMembers
+        if(filterString === ''){
+            dispatch(setFilteredMembers(state.members))
+        }
+        else{
+            dispatch(setFilteredMembers(
+                state.members.filter(member=>(
+                    new RegExp(`${filterString}`,'i').test(member.name)
```
Comment (l2silver) [inline web-client/src/shared/redux/volunnteer-toronto/members/index.ts:99]: This is excellent. In the future we might want to switch this out for some kind of fuzzy search library, but this is perfect for now

**Review by l2silver — APPROVED** [review]
This is perfect. The only thing we need to do is create a separate tickets for tests, and add them in.

Really great work!

**Comment by Haris188** [comment]
Hey @l2silver ,

Thank you very much for the review. So is it okay to merge this pull request now? And also test ticket for this would be in next sprint or this one? And would I do that ticket?

**Comment by l2silver** [comment]
> Hey @l2silver ,
> 
> Thank you very much for the review. So is it okay to merge this pull request now? And also test ticket for this would be in next sprint or this one? And would I do that ticket?

@Haris188 Yeah, merge it in. If you have time, feel free to add it to the current sprint. But you've already done a lot this week, so i'll leave that up to you. I would prefer if you handled the tests though. Thanks

**Comment by Haris188** [comment]
> > Hey @l2silver ,
> > Thank you very much for the review. So is it okay to merge this pull request now? And also test ticket for this would be in next sprint or this one? And would I do that ticket?
> 
> @Haris188 Yeah, merge it in. If you have time, feel free to add it to the current sprint. But you've already done a lot this week, so i'll leave that up to you. I would prefer if you handled the tests though. Thanks

I will Create the ticket and assign it to myself, but I will do it in next sprint. :)

**Pull request merged**